src/rules.jl

```diff
@@ -546,10 +549,32 @@ function lrp!(Rᵏ, rule, layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)
     # lrp pass
     ## Rᵏ⁺¹ ->(scale) Rᵏ ->(normalize) Rᵏ
     ## scale: call LRP on affine layer with "subrule" rule.affine_rule
-    lrp!(Rᵏ, rule, layer.diag, modify_layer(rule, layer.diag), aᵏₙ, Rᵏ⁺¹)
     ## normalize
     s = @. Rᵏ / stabilize_denom(z, LRP_DEFAULT_STABILIZER)
     Rᵏ .= aᵏ .* (s .- mean(s, dims=n_dims))
+end 
+
+function lrp!(Rᵏ, ::LayerNormRule, layer::LayerNorm{F,D}, _modified_layer, aᵏ, Rᵏ⁺¹) where {F,D<:typeof(identity)}
+    n_dims = 1:length(layer.size)
+    μₐ = mean(aᵏ; dims=n_dims)
+    s = @. Rᵏ⁺¹ / stabilize_denom(aᵏ - μₐ, LRP_DEFAULT_STABILIZER)
+    μₛ = mean(s; dims=n_dims)
+    @. Rᵏ = aᵏ * (s - μₛ)
+end
+
+function lrp!(Rᵏ, ::LayerNormRule, layer::LayerNorm, _modified_layer, aᵏ, Rᵏ⁺¹)
```

Comment:
It looks to me like this method has the same argument types as the one on L539.
I'm guessing the other one can be removed?

Reply:
Yes, sorry, I think the methods should now be correct.